client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -531,6 +532,20 @@ export const confirmPackages = ( orderId, siteId ) => ( dispatch, getState ) =>
 	tryGetLabelRates( orderId, siteId, dispatch, getState );
 };
 
+export const saveCustoms = ( orderId, siteId ) => {
```

Comment:
I'm not sure I understand what this action is for (even after reading the subsequent PR). In this PR, it looks like it prevents users from submitting an incorrect customs form after a customs form was submitted at least once.

Reply:
> In this PR, it looks like it prevents users from submitting an incorrect customs form after a customs form was submitted at least once.

Exactly. This is made to support this UX:
* At the beginning, the Customs step is not "valid", but also it doesn't give the user any errors right away because that would be too aggresive.
* When the user clicks the "Confirm" button, then the whole step is validated.
* At that moment, if there are invalid fields in the step, the errors will be shown to the user.

So the `submitted` boolean flag is to simulate that "user has already tried to save the step, from now own perform full validations". It's not a simple "submitted" flag in the real UI PR, but I needed something here to simulate that behaviour.

Reply:
Thanks for the details :)